src/java/org/apache/cassandra/db/SystemKeyspace.java

```diff
@@ -1242,7 +1250,7 @@ public static UUID getSchemaVersion()
         String req = "SELECT schema_version FROM system.%s WHERE key='%s'";
         UntypedResultSet result = executeInternal(format(req, LOCAL, LOCAL));
 
-        if (!result.isEmpty() && result.one().has("schema_version"))
+        if (result != null && result.isEmpty() && result.one().has("schema_version"))
```

Comment:
`... && !result.isEmpty()` ?

Reply:
thanks! good catch.

Reply:
I will revert this change. I was thinking it will be an easy fix but it seems like the whole class is filled with issues like this. We may dedicate separate ticket for that to clean it up properly.